### PR TITLE
Bug 1276213 - switching between containers doesnt connect to the right container

### PR DESCRIPTION
**Pull request by jwforres**

### CODE REVIEW

**Comment by jwforres** [comment]
@benjaminapetersen  PTAL

**Comment by jwforres** [comment]
@spadgett

File: assets/app/scripts/controllers/pod.js [hunk 1]
```diff
@@ -97,12 +97,17 @@ angular.module('openshiftConsole')
...
+          canInitAgain: false
         });
 
+        if (!$scope.options || !$scope.options.container) {
+          angular.extend($scope, {
+            options: {
+              container: $scope.pod.spec.containers[0].name
+            }
+          });
+        }
```
Comment (spadgett) [inline assets/app/scripts/controllers/pod.js]: Is it possible to initialize this once when we call DataService.get() for the pod?

**Comment by jwforres** [comment]
closing since this is going to change with @spadgett's refactoring changes to logs for the perf issues

**Comment by spadgett** [comment]
Fixed by #5537

**Pull request closed without merging**